mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -322,7 +333,7 @@ abstract class RecommendationTests(sc: SparkContext) extends PerfTest {
     val implicitRatings: Boolean = booleanOptionValue(IMPLICIT)
 
     val data = DataGenerator.generateRatings(sc, numUsers, numProducts,
-      math.ceil(numRatings * 1.25).toLong, implicitRatings,numPartitions,seed)
+      math.ceil(numRatings).toLong, implicitRatings,numPartitions,seed)
```

Comment:
I'll go ahead and make this change for previous versions since it seems like the one item needed.